ui/src/api-logic/graphql/queries/get-repos.query.ts

```diff
@@ -1,20 +1,35 @@
 import { gql } from '@apollo/client'
 
-const GET_REPOS = gql`
-  query getRepos($search: String!, $first: Int, $offset: Int) {
-    serviceAuthCredentials(filter: {type: {equalTo: "GITHUB_PAT"}}) {
+const GET_ALL_REPOS = gql`
```

Comment:
```suggestion
const GET_REPO_TOTAL_COUNT = gql`
```

Just to be a bit clearer